Description: Try to fix "AttributeError: object has no attribute 'summary_detail'"
 Feedparser in Debian is much more recent than the embedded copy, so try to adjust
Author: Olivier Berger <obergix@debian.org>

--- planet-venus-0~git9de2109.orig/planet/reconstitute.py
+++ planet-venus-0~git9de2109/planet/reconstitute.py
@@ -76,9 +74,8 @@ def id(xentry, entry):
     elif entry.has_key("title") and entry.title:
         entry_id = (entry.title_detail.base + "/" +
             md5(entry.title).hexdigest())
-    elif entry.has_key("summary") and entry.summary:
-        entry_id = (entry.summary_detail.base + "/" +
-            md5(entry.summary).hexdigest())
+    elif entry.has_key("summary") and entry.summary and entry.has_key("summary_detail") and entry.summary_detail:
+      entry_id = (entry.summary_detail.base + "/" + md5(entry.summary).hexdigest())
     elif entry.has_key("content") and entry.content:
 
         entry_id = (entry.content[0].base + "/" + 
